examples/mobile/UIComponents/components/appbar/navigateUpButton.html

```diff
@@ -16,12 +16,12 @@
 				<button class="ui-btn ui-btn-icon ui-btn-icon-back" data-rel="back"></button>
```

Comment:
I think buttons in appBar are more similar to "flat" button than "contained". Contained button dosn't have icons.
Try add data-style="flat" to buttons in appBar.

Reply:
You can check also this change: https://github.sec.samsung.net/HighPerformanceWeb/TAU/pull/341/commits/aa6e631b1ad3e724c790e1c9fb4a5340a756fda3#diff-a02df192dc819d403358047ff962b511

Reply:
Ok, thanks for the hint. I will apply flat button to appbar.